# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/cookiebanners/nsICookieBannerTelemetryService.idl
# Commit: 40e5fa7c8b85
# Full Hash: 40e5fa7c8b85cc8044b0f60e249d7730817e33ab
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2024-03-30 09:10:19
# Regressor Bug: 1874741
# File Overlap Count: 1
# Description:
#   Bug 1874741 - Part 3: Implement the telemetry to collect GDPR choice on Google Search. r=anti-tracking-reviewers,pbz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205443
# ==============================================================================

diff -r 67c3461e1556 -r 40e5fa7c8b85 toolkit/components/cookiebanners/nsICookieBannerTelemetryService.idl
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/toolkit/components/cookiebanners/nsICookieBannerTelemetryService.idl	Fri Mar 29 23:15:12 2024 +0000
@@ -0,0 +1,13 @@
+/* -*- Mode: C++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "nsISupports.idl"
+
+/**
+ * Service singleton which collect cookie banner telemetry.
+ */
+[scriptable, builtinclass, uuid(56197e18-d144-45b5-9f77-84102f064462)]
+interface nsICookieBannerTelemetryService : nsISupports {};
